tools/c7n_azure/c7n_azure/resources/appserviceplan.py

```diff
@@ -118,6 +118,7 @@ class ResizePlan(AzureBaseAction):
 
     schema = utils.type_schema(
         'resize-plan',
+        required=['size'],
```

Comment:
Do we want/need to do this as opposed to populating it from the resource and requiring `oneOf: size, count`?

Seems like a good scenario to be able to go reduce instance counts as part of a policy (regardless of SKU).